src/solids4FoamModels/materialModels/mechanicalModel/mechanicalLaws/nonLinearGeometryLaws/neoHookeanElasticMisesPlastic/neoHookeanElasticMisesPlastic.C

```diff
@@ -773,24 +773,22 @@ void Foam::neoHookeanElasticMisesPlastic::calculateStress
     // Calculate deviatoric stress
     const surfaceSymmTensorField s(sTrial - 2*mu_*DEpsilonPf_);
 
-    //Update bEbar
-    if (updateBEbarConsistent_)
-    {
-        const surfaceSymmTensorField devBEbar(s/mu_);
-        bEbarf_ = devBEbar + this->Ibar(devBEbar)*I;
-    }
-    else
-    {
-        bEbarf_ = (s/mu_) + Ibar*I;
-    }
+//    //Update bEbar
```

Comment:
@federico081997: is there a reason you have disabled the update of `bEbarf_`? I guess the procedure will not be correct for plasticity with this commented.